filament/backend/src/metal/MetalState.h

```diff
@@ -221,7 +221,8 @@ struct PipelineState {
     NSUInteger sampleCount = 1;                                                // 8 bytes
     BlendState blendState;                                                     // 56 bytes
     bool colorWrite = true;                                                    // 1 byte
-    char padding[7] = { 0 };                                                   // 7 bytes
+    bool stencilWrite = false;                                                 // 1 byte
```

Comment:
I find it brave from Filament developers to assume that `sizeof(bool)` is always 1. That's not guaranteed at all, is it? Anyway if `colorWrite` worked so far, so should your addition. Just strange...

Reply:
Yes MSVC agreed after version 5 or sth ([Stackoverflow answer](https://stackoverflow.com/a/6935286))